app/src/main/java/org/jellyfin/androidtv/ui/ExpandableTextView.java

```diff
@@ -4,22 +4,23 @@
 import android.content.Intent;
 import android.graphics.Canvas;
 import android.graphics.Rect;
-import androidx.core.app.ActivityOptionsCompat;
-import androidx.appcompat.widget.AppCompatTextView;
 import android.text.Layout;
 import android.util.AttributeSet;
 import android.view.SoundEffectConstants;
 import android.view.View;
+import android.widget.TextView;
 
 import org.jellyfin.androidtv.R;
 import org.jellyfin.androidtv.TvApp;
 import org.jellyfin.androidtv.details.ExpandedTextActivity;
 
+import androidx.core.app.ActivityOptionsCompat;
+
 /**
  * Created by spam on 9/30/2016.
  */
 
-public class ExpandableTextView extends AppCompatTextView {
+public class ExpandableTextView extends TextView {
```

Comment:
Why are we moving away from using the Androidx appcompat classes?

Reply:
There is only a few custom views that use the AppCompat classes. However if we truly used AppCompat we would need to change almost all UI related code (XML and Java) to use AppCompat.

The views we did use AppCompat for did not really use any features that were not available in the first versions of Android so it was not necessary to use it.

We can always add back AppCompat in the future but it would need to be done properly for all java/xml. You can see what I mean by using the "Migrate to Appcompat..." function in Android Studio. It gives you an impression of how much code was not AppCompat yet.